src/cnab.rs

```diff
@@ -1,8 +1,5 @@
-use semver::Version;
-use serde::{Deserialize, Serialize};
-use std::collections::HashMap;
+use std::collections::btree_map::BTreeMap;
```

Comment:
To preserve the order of the objects in maps, this PR switches all maps from HashMap to BTreeMap.